src/VelocityUKF.cpp

```diff
@@ -3,7 +3,7 @@
 using namespace uwv_kalman_filters;
 
 const std::string VelocityUKF::acceleration_measurement = "acceleration";
```

Comment:
The filters implemented in this library should on the long run not implement the interfaces of the pose estimation library. In contrast they should purely relay on MTK and should have a fixed and known set of inputs and states.
The reason why the VelocityProvider currently implements this interface is mainly because the filter was a test. And for testing the interface is quite nice.
Ideally the filters in this library have only a very thin layer to MTK. This should also for the future pose estimator be the case. Since the VelocityProvider was now tested I can clean it up, but that won't be part of this PR.